test/e2e/node/security_context.go

```diff
@@ -146,6 +217,31 @@ var _ = SIGDescribe("Security Context", func() {
 		testPodSELinuxLabeling(f, false, true)
 	})
 
+	ginkgo.It("should support seccomp localhost profile for a container", func() {
+		ginkgo.By("Preparing the nodes")
+		profileName := fmt.Sprintf("test-%v.json", string(uuid.NewUUID()))
+		spec := setupLocalhostSeccompProfile([]seccompProfile{
+			{
+				name: profileName,
+				json: `{"defaultAction": "SCMP_ACT_ERRNO"}`,
```

Comment:
I think by blocking all calls this may lead to some unexpected behaviours at times, as highlighted on https://github.com/kubernetes/kubernetes/issues/85191. I would probably use "SCMP_ACT_ALLOW" instead - which effectively is the same as the current seccomp default (`unconfined`).

Reply:
Yeah I think I will make the test-case a bit explicit by blocking only one syscall like `mkdir` and testing for that within the container.